sdk/python/kfp/filter_pb2.py

```diff
@@ -0,0 +1,409 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
How did you generate this file? Would you mind adding related commands in https://github.com/kubeflow/pipelines/blob/master/backend/api/build_kfp_server_api_python_package.sh? and let this be part of the `kfp-server-api` package, because is this generated from backend api.